virtio-queue/src/descriptor_utils.rs

```diff
@@ -0,0 +1,850 @@
+// Copyright 2019 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
We currently claim dual licensing of Apache + BSD. So this probably requires us to tweak the documentation and Cargo.toml if that is no longer universally true?

Reply:
Hello, how the documentation shall be tweaked? Do you think I need to add something to the PR?

Reply:
If the license of virtio-queue changed, it would at least have to be reflected in the Cargo.toml file. Probably the `OR` there has to become and `AND`?

Reply:
Is changing the `OR` to `AND` enough to fix the licensing? Is this change needed at all?

Reply:
I reverted the changes and submit https://github.com/rust-vmm/vm-virtio/issues/283.